bundles/org.eclipse.ui.ide/schema/markerSupport.exsd

```diff
@@ -192,6 +192,24 @@ ON_ANY_IN_SAME_CONTAINER: on any item with the same top level container as the s
                </appinfo>
             </annotation>
          </attribute>
+         <attribute name="application">
+            <annotation>
+               <documentation>
+                  The application attribute describes how the reference should be applied.
```

Comment:
Is the attribute optional? What is the default value that is used when the attribute is not set? please document.
This is probably an API change, which requires multiple committers to agree upon. good luck.